src/sentry/integrations/utils/code_mapping.py

```diff
@@ -155,45 +156,15 @@ def __init__(self, frame_file_path: str) -> None:
         self.extension = get_extension(frame_file_path)
         if not self.extension:
             raise UnsupportedFrameFilename("It needs an extension.")
-        if self.frame_type() == "packaged":
-            self._packaged_logic(frame_file_path)
-        else:
-            self._straight_path_logic(frame_file_path)
-
-    def frame_type(self) -> str:
-        type = "packaged"
-        if self.extension not in ["py"]:
-            type = "straight_path"
-        return type
```

Comment:
Here is where we erroneously declared a stack trace as "packaged" if it is a python file. You can see in many places where we incorrectly branch the logic based on this assumption